lib/bloc/podcast_payments/podcast_payments_bloc.dart

```diff
@@ -263,9 +267,10 @@ class PodcastPaymentsBloc with AsyncActionsHandler {
           log.info("succesfully paid $netPay to destination ${d.address}");
 
           _addToPodcastHistory(
-              podcastId: episode.metadata["feed"]["id"],
+              podcastId: episode.metadata["feed"]["id"].toString(),
               podcastName: episode.metadata["feed"]["title"],
               podcastImageUrl: episode.metadata["feed"]["image"],
+              podcastUrl: episode.metadata["feed"]["originalUrl"],
               satsSpent: total,
```

Comment:
shouldn't we use here `netPay` instead of `total` ?

Reply:
Yes, changed it to `netPay`